internal/cli/kraft/cloud/deploy/deploy.go

```diff
@@ -246,29 +251,80 @@ func (opts *DeployOptions) Run(ctx context.Context, args []string) error {
 						kraftcloud.WithDefaultMetro(opts.Metro),
 					)
 
-					var oldInsts []instances.GetResponseItem
-					if utils.IsUUID(opts.Rollout) {
-						oldInsts, err = instanceClient.GetByUUIDs(ctx, opts.Rollout)
+					serviceClient := kraftcloud.NewServicesClient(
```

Comment:
The client is already initialized and readily available with `opts.Client.Services()`.

Reply:
(Same comment for `instanceClient`, but that code predates your changes)